TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php

```diff
@@ -189,7 +191,7 @@ public function cleanCommand()
                             }
                         }
                     }
-                    $this->outputLine('Removed %s resource object(s) from the database.', array(count($brokenResources)));
+                    $this->outputLine('Removed %s resource object(s) from the database.', [count($brokenResources)]);
                 break;
```

Comment:
Uhm, I think it was a display error in the diff view, now iwth the comments it looks normal so nevermind. It looked like the rest of the cases was deleted as well....